wavetool_copycat/wavetool_copycat.py

```diff
@@ -0,0 +1,363 @@
+		self.LED_2_scale_factor_option_menu = tk.OptionMenu(self.frame_LED_2_current, self.default_options_LED_2_scale_factor, *options_LED_2_scale_factor)
+		self.LED_2_scale_factor_option_menu.grid(row = 1, column = 1)
+
+		self.LED_2_scale = tk.Scale(self.frame_LED_2_current, orient=tk.HORIZONTAL)
+		self.LED_2_scale.grid(row = 2 , column = 0)
+
+		self.frame_LED_2_final_I_LED_field = tk.LabelFrame(self.frame_LED_2_current, bd = 0)
+		self.frame_LED_2_final_I_LED_field.grid(row = 3 , column = 0)
+		self.LED_2_final_I_LED_text = tk.Label(self.frame_LED_2_final_I_LED_field, text = "Final I_LED")
+		self.LED_2_final_I_LED_text.grid(row = 0, column = 0)
+		self.LED_2_final_I_LED_entry = tk.Entry(self.frame_LED_2_final_I_LED_field)
+		self.LED_2_final_I_LED_entry.grid(row = 0, column = 1)
+		self.LED_2_mA_text = tk.Label(self.frame_LED_2_final_I_LED_field, text = "mA")
+		self.LED_2_mA_text.grid(row = 0, column = 2)
+
+	def frame_LED_3_current(self, master):
+		self.frame_LED_3_current = tk.LabelFrame(master, bd = 2, text = "LED 3 Current")
+		self.frame_LED_3_current.grid(row = 0, column = 2)
+
+		self.LED_3_I_LED_coarse_text = tk.Label(self.frame_LED_3_current, text = "Scale Factor")
+		self.LED_3_I_LED_coarse_text.grid(row = 0, column = 0)
+		options_LED_3_I_LED_coarse = ('X', 'Y', 'Z')
+		self.default_options_LED_3_I_LED_coarse= tk.StringVar()
+		self.default_options_LED_3_I_LED_coarse.set(options_LED_3_I_LED_coarse[0])
+		self.LED_3_I_LED_coarse_option_menu = tk.OptionMenu(self.frame_LED_3_current, self.default_options_LED_3_I_LED_coarse, *options_LED_3_I_LED_coarse)
+		self.LED_3_I_LED_coarse_option_menu.grid(row = 0, column = 1)
+
+		self.LED_3_scale_factor_text = tk.Label(self.frame_LED_3_current, text = "Scale Factor")
+		self.LED_3_scale_factor_text.grid(row = 1, column = 0)
+		options_LED_3_scale_factor = ('X', 'Y', 'Z')
+		self.default_options_LED_3_scale_factor= tk.StringVar()
+		self.default_options_LED_3_scale_factor.set(options_LED_3_scale_factor[0])
+		self.LED_3_scale_factor_option_menu = tk.OptionMenu(self.frame_LED_3_current, self.default_options_LED_3_scale_factor, *options_LED_3_scale_factor)
+		self.LED_3_scale_factor_option_menu.grid(row = 1, column = 1)
+
+		self.LED_3_scale = tk.Scale(self.frame_LED_3_current, orient=tk.HORIZONTAL)
+		self.LED_3_scale.grid(row = 2 , column = 0)
+
+		self.frame_LED_3_final_I_LED_field = tk.LabelFrame(self.frame_LED_3_current, bd = 0)
+		self.frame_LED_3_final_I_LED_field.grid(row = 3 , column = 0)
+		self.LED_3_final_I_LED_text = tk.Label(self.frame_LED_3_final_I_LED_field, text = "Final I_LED")
+		self.LED_3_final_I_LED_text.grid(row = 0, column = 0)
+		self.LED_3_final_I_LED_entry = tk.Entry(self.frame_LED_3_final_I_LED_field)
+		self.LED_3_final_I_LED_entry.grid(row = 0, column = 1)
+		self.LED_3_mA_text = tk.Label(self.frame_LED_3_final_I_LED_field, text = "mA")
+		self.LED_3_mA_text.grid(row = 0, column = 2, sticky = tk.E)
+
```

Comment:
It might be good if we use  __name__ == '__main__' and put the next four lines of code under a main function.